src/CalculatorUITestFramework/StandardAoTCalculatorPage.cs

```diff
@@ -0,0 +1,201 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Interactions;
+using System;
+using System.Drawing;
+
+namespace CalculatorUITestFramework
+{
+    /// <summary>
+    /// This class contains the UI automation objects and helper methods available when the Calculator is in Standard / AoT Mode.
+    /// </summary>
+    public class StandardAoTCalculatorPage
+    {
+        private WindowsDriver<WindowsElement> session => WinAppDriver.Instance.CalculatorSession;
+        public GlobalCalculatorUI GlobalCalculatorUI = new GlobalCalculatorUI();
+        public StandardOperatorsPanel StandardOperators = new StandardOperatorsPanel();
+        public NavigationMenu NavigationMenu = new NavigationMenu();
+        public WindowsElement EnterAlwaysOnTopButton => this.session.TryFindElementByAccessibilityId("NormalAlwaysOnTopButton");
+        public WindowsElement ExitAlwaysOnTopButton => this.session.TryFindElementByAccessibilityId("ExitAlwaysOnTopButton");
+        public AppiumWebElement ToolTip => WinAppDriver.Instance.CalculatorSession.FindElementByClassName("ToolTip").FindElementByClassName("TextBlock");
+
+        public void StandardNonAoTMode()
+        {
+            string sourceF = this.session.PageSource;
+            if (sourceF.Contains("ExitAlwaysOnTopButton"))
+            {
+                this.ExitAlwaysOnTopButton.Click();
+                Assert.AreEqual("Standard", this.GlobalCalculatorUI.GetCalculatorHeaderText());
+            }
+            else
+            {
+                string source2 = this.session.PageSource;
+                if (source2.Contains("NormalAlwaysOnTopButton"))
+                {
+                    return;
+                }
+                else
+                {
+                    throw new NotFoundException("Could not find the 'Keep on top' button");
+                }
+            }
+        }
+        ///// <summary>
+        ///// Ensures the calculator is in AoT(Keep on top) mode, and verifies that the AoT mode through the absence of the header
+        ///// </summary>
+        public void StandardAoTMode()
+        {
+            string sourceG = this.session.PageSource;
+            if (sourceG.Contains("NormalAlwaysOnTopButton"))
+            {
+                this.EnterAlwaysOnTopButton.Click();
+                this.ExitAlwaysOnTopButton.WaitForDisplayed();
+                string sourceH = WinAppDriver.Instance.CalculatorSession.PageSource;
+                if (sourceH.Contains("Header"))
+                {
+                    throw new NotFoundException("Failed to enter 'Keep on top' mode; In AoT mode, Calculator does not have header");
+                }
+            }
+        }
+        /// <summary>
+        /// Checks to see if the size of the Calculator window is within the AoT arange of size
+        /// </summary>
+        public void AoTWindowSizeWithinRange()
```

Comment:
Is testing our implementation or just the compact overlay api?

Reply:
This is test framework method that helps to verify that the calculator is in AoT mode.